app/views/support_interface/application_forms/_application_navigation.html.erb

```diff
@@ -15,9 +15,27 @@
   ].concat(title != 'Details' ? [{ text: title }] : [])) %>
 <% end %>
 
-<p class="govuk-body govuk-!-margin-bottom-6">
-  <%= govuk_link_to 'Emails about this application', support_interface_email_log_path(application_form_id: @application_form.id), class: 'govuk-link--no-visited-state' %>
-</p>
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <span class="govuk-caption-l"><%= @application_form.support_reference %></span>
+    <h1 class="govuk-heading-l"><%= break_email_address(@application_form.candidate.email_address) %></h1>
+    <% unless HostingEnvironment.production? %>
+      <%= govuk_button_to 'Sign in as this candidate', support_interface_impersonate_candidate_path(@application_form.candidate) %>
+    <% end %>
+  </div>
+
+  <div class="govuk-grid-column-one-third">
+    <div class="govuk-inset-text app-inset-text--important app-inset-text--narrow-border govuk-!-margin-top-0">
+      <h2 class="govuk-heading-s govuk-!-margin-bottom-2">Useful links for this application</h2>
+      <ul class="govuk-list govuk-!-font-size-16">
+        <li><%= govuk_link_to 'Emails sent', support_interface_email_log_path(application_form_id: @application_form.id), class: 'govuk-link--no-visited-state' %></li>
+        <li><%= govuk_link_to 'Logit logs', '#', class: 'govuk-link--no-visited-state' %></li>
+        <li><%= govuk_link_to 'Sentry errors', '#', class: 'govuk-link--no-visited-state' %></li>
```

Comment:
Support agents do not have access to neither sentry or logit, and they should not be expected to understand those errors either, so I don't think there is value in adding these two links.

Reply:
These links are in the trello card in the support feature wish list… although I can’t see who originally wrote the card.

Reply:
Good point though… maybe we should only include internal links?

Reply:
I wrote it! The idea was to have links that were useful for developers in a "Useful links for developers" section